Makefile

```diff
@@ -150,7 +151,7 @@ build-image: $(DOCKERFILE_DEVEL)
 		--build-arg MOQ_VERSION="$(MOQ_VERSION)" \
 		--tag $(BUILDIMAGE) \
 		-f $(DOCKERFILE_DEVEL) \
-		.
+		$(K8S_TEST_INFRA)
```

Comment:
This variable will be replaced as `"https://github.com/NVIDIA/k8s-test-infra.git"`.
Is that a valid context for docker?

Reply:
yes, Docker supports remote Dockerfiles in Git repos

Reply:
Ack